chcemvediet/apps/accounts/forms.py

```diff
@@ -61,3 +61,14 @@ def signup(self, request, user):
         user.profile.city = self.cleaned_data[u'city']
         user.profile.zip = self.cleaned_data[u'zip']
         user.profile.save()
+
+class SettingsForm(forms.Form):
+
+    anonymized_inforequest = forms.BooleanField(
+        label=_(u'accounts:SettingsForm:anonymized_inforequest:label'),
+        required=False,
+    )
+
+    def __init__(self, user, *args, **kwargs):
```

Comment:
@martinmacko47 tymto si niesom isty. Moze to byt riesene tak, ze som pridal argument pre formular?

Reply:
@viliambalaz Lepsie by bolo, aby parameter `initial` sa doplnil v `__init__` metode formularu. Kedze `user` objekt ma k dispozicii, tak to moze doplnit tam:

```
kwargs[u'initial'] = {u'anonymize_inforequests': request.user.profile.anonymize_inforequests}
super(SettingsForm, self).__init__(*args, **kwargs)
```

Tj. aby view neriesilo, ake fieldy form pouziva. Aby to vsetko spravil samotny form. Ze ked sa prida novy field, tak sa prida len na jedno miesto - do formu.

Reply:
Ok, prvykrat som to zle pochopil, dal som argument `initial` to priamo ako parameter. Skuskal som aj moznost `forms.ModelForm`, ale s klasickou `forms.Form` sa lepsie pracovalo a jednoduchsie som si ju vedel prisposobit. dakujem